### PR TITLE
build: Add headers to CMake project files

### PR DESCRIPTION
**Pull request by orgads**
Useful for listing them in the IDE project tree, for IDEs that use CMake file api (like Qt Creator).

### CODE REVIEW

File: src/storage/CMakeLists.txt [hunk 1]
```diff
@@ -4,6 +4,8 @@ add_subdirectory(remote)
 set(
   sources
   Storage.cpp
+  Storage.hpp
+  types.hpp
```
Comment (jrosdahl) [inline src/storage/CMakeLists.txt:8]: Would it make sense to glob these as well?

Comment (orgads) [reply src/storage/CMakeLists.txt:8]: Sure.

Comment (orgads) [reply src/storage/CMakeLists.txt:8]: Done

**Comment by jrosdahl** [comment]
Thanks.

**Pull request merged**